### PR TITLE
[CN-615] Split restore and restore_local commands

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

File: api/v1alpha1/hazelcast_types.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ type HazelcastSpec struct {
 
 	// B&R Agent configurations
 	// +optional
-	// +kubebuilder:default:={repository: "docker.io/hazelcast/platform-operator-agent", version: "0.1.10"}
+	// +kubebuilder:default:={repository: "docker.io/kutluhanhazelcast/platform-operator-agent", version: "0.1.8"}
```
Comment (SeriyBg) [inline api/v1alpha1/hazelcast_types.go:102]: Seems to be a leftover

**Review by SeriyBg — COMMENTED** [review]
LGTM, just some leftover repo name needs be reverted 😃

**Comment by mtyazici** [comment]
Should we update `github.com/hazelcast/platform-operator-agent v0.1.10` in `go.mod` file after we merge the agent side PR? We will eventually need to update the agent version in the `go.mod` file and we know that the import paths will change after we merge agent side PR.

**Comment by kutluhanmetin** [comment]
> github.com/hazelcast/platform-operator-agent

**Review by SeriyBg — APPROVED** [review]

**Review by mtyazici — APPROVED** [review]

**Pull request merged**